CHANGELOG.md

```diff
@@ -48,6 +48,7 @@
 - The command `operator-sdk generate k8s` no longer requires users to explicitly set GOROOT in their environment. Now, GOROOT is detected using `go env GOROOT` and set automatically. ([#2754](https://github.com/operator-framework/operator-sdk/pull/2754))
 - `operator-sdk generate csv` and `operator-sdk test local` now parse multi-manifest files correctly. ([#2758](https://github.com/operator-framework/operator-sdk/pull/2758))
 - Fixed CRD validation generation issue with `status.Conditions`. ([#2739](https://github.com/operator-framework/operator-sdk/pull/2739))
+- Fixed helm-operator issue with reconciliation of arrays ([#2777](https://github.com/operator-framework/operator-sdk/pull/2777))
```

Comment:
```suggestion
- Fix issue faced in the reconciliation of helm based-operators.  ([#2777](https://github.com/operator-framework/operator-sdk/pull/2777))
```

Reply:
This does not only apply to additional containers, every array was affected by this, like env variables, volumeMounts, ...

Reply:
Somthing such as above. I updated it. WDYT?